src/js/plugins/plugin.item_nav.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * plugin to load BookReaderTemplate
+ */
+import { BookReaderTemplate } from "../../BookReaderTemplate/template";
+
+window.customElements.define('bookreader-template', BookReaderTemplate);
```

Comment:
I was initially thinking we might want `BookReaderTemplate` constructed/initialized from within this file; and use the `BookReader.prototype.init` as a hook. That would let us support the current "interface" of `<div id="BookReader" />`. I think our main goal was to have it so that petabox could use bookreader "binaries" (everything in the BookReader folder), _but_ avoid using item navigator.

I think we're having trouble predicting the future on this one and getting stuck; let's run with this as is, and see what we need to change :+1:

Reply:
resolving for now, we can get back to this in the future